drivers/charger/charger_handlers.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/syscall_handler.h>
+#include <zephyr/drivers/charger.h>
+
+static inline int z_vrfy_charger_get_prop(const struct device *dev, const charger_prop_t prop,
+					  union charger_propval *val)
+{
+	union charger_propval k_val = {0};
+
+	Z_OOPS(Z_SYSCALL_DRIVER_CHARGER(dev, get_property));
+
+	Z_OOPS(z_user_from_copy(&k_val, val, sizeof(union charger_propval)));
```

Comment:
is this needed? it's not like you expect `val` to be initialized anyway

Reply:
ack